lib/ReactViewModels/MouseCoords.js

```diff
@@ -127,26 +134,12 @@ export default class MouseCoords {
         }
     }
 
-    debounceSampleAccurateHeight(globe, position) {
-        // After a delay with no mouse movement, get a more accurate height.
-        Cartographic.clone(position, this.lastHeightSamplePosition);
-
-        const terrainProvider = globe.terrainProvider;
-        if (!(terrainProvider instanceof EllipsoidTerrainProvider)) {
-            clearTimeout(this.accurateHeightTimer);
-            this.accurateHeightTimer = setTimeout(()=> {
-                this.sampleAccurateHeight(terrainProvider, position);
-            }, this.accurateSamplingDebounceTime);
-        }
-    }
 
     sampleAccurateHeight(terrainProvider, position) {
-        this.accurateHeightTimer = undefined;
         if (this.tileRequestInFlight) {
             // A tile request is already in flight, so reschedule for later.
-            this.accurateHeightTimer = setTimeout(()=> {
-                this.sampleAccurateHeight(terrainProvider, position);
-            }, this.accurateSamplingDebounceTime);
+            this.debounceSampleAccurateHeight.cancel();
```

Comment:
This works, and looks good to me! However I haven't quite got my head around this part, so I'll leave it to @kring to review again.